### ISSUE TITLE
Values for alpha and beta

### ISSUE THREAD

**Issue by Alexethz** [W-qp/MultiSenseSeg#1]
Thank you for your impressive work! I wonder what are the recommended values for alpha and beta that are used to compute the auxiliary loss. I think they are not provided in the paper.

**Comment by W-qp** [comment]
Thank u for your interest in our work. The Alpha and Beta values I used during 100 iterations were 0.5 and 0.4, respectively. These hyperparameters are related to the training epochs. I recommend that Beta should not be lower than 0.3, and while I haven't tried many values for Alpha, it shouldn't be too high.
